packages/venia-ui/lib/components/CheckoutPage/PaymentInformation/paymentMethods.js

```diff
@@ -36,37 +36,48 @@ const PaymentMethods = props => {
         return null;
     }
 
-    const radios = availablePaymentMethods.map(({ code, title }) => {
-        // If we don't have an implementation for a method type, ignore it.
-        if (!Object.keys(payments).includes(code)) {
-            return;
-        }
+    const radios = availablePaymentMethods
+        .map(({ code, title }) => {
+            // If we don't have an implementation for a method type, ignore it.
+            if (!Object.keys(payments).includes(code)) {
+                return;
+            }
 
-        const isSelected = currentSelectedPaymentMethod === code;
-        const PaymentMethodComponent = payments[code];
-        const renderedComponent = isSelected ? (
-            <PaymentMethodComponent
-                onPaymentSuccess={onPaymentSuccess}
-                onPaymentError={onPaymentError}
-                resetShouldSubmit={resetShouldSubmit}
-                shouldSubmit={shouldSubmit}
-            />
-        ) : null;
-
-        return (
-            <div key={code} className={classes.payment_method}>
-                <Radio
-                    label={title}
-                    value={code}
-                    classes={{
-                        label: classes.radio_label
-                    }}
-                    checked={isSelected}
+            const isSelected = currentSelectedPaymentMethod === code;
+            const PaymentMethodComponent = payments[code];
+            const renderedComponent = isSelected ? (
+                <PaymentMethodComponent
+                    onPaymentSuccess={onPaymentSuccess}
+                    onPaymentError={onPaymentError}
+                    resetShouldSubmit={resetShouldSubmit}
+                    shouldSubmit={shouldSubmit}
                 />
-                {renderedComponent}
-            </div>
-        );
-    });
+            ) : null;
+
+            return (
+                <div key={code} className={classes.payment_method}>
+                    <Radio
+                        label={title}
+                        value={code}
+                        classes={{
+                            label: classes.radio_label
+                        }}
+                        checked={isSelected}
+                    />
+                    {renderedComponent}
+                </div>
+            );
+        })
+        .filter(paymentMethod => !!paymentMethod);
+
+    const noPaymentMethodMessage = !radios.length ? (
+        <span className={classes.no_payment_method}>
+            {formatMessage({
```

Comment:
You'll need to make sure this method is available.

```es6
import { useIntl } from 'react-intl';

....

const { formatMessage } = useIntl();
```

Reply:
Added.